src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -123,11 +123,16 @@ def is_multi_asic():
 
 
 def get_asic_id_from_name(asic_name):
+    """
+    Get the asic id from the asic name for multi-asic platforms
```

Comment:
What happens if this function is called on a single-ASIC platform? Might want to add details here.

Reply:
Added the details if this API is called on a single-ASIC platform.